src/resources/js/script.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file This file contains all JavaScript specific to Schedule Day View.
+ * This file should load after all vendor and core events JavaScript.
+ * @version 1.0.0
+ */
+
+(function ( window, document, $, td, te, tf, ts, tt, config, dbug ) {
```

Comment:
Guessing this mimics existing code and it's not essential to change it, but it would be cleaner to drop the params we aren't using (most of these aren't actually used or else don't need to be passed in in the first place).